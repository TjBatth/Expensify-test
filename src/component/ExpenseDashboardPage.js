import React from 'react'
import ExpesneList from './ExpenseList'
import EXpenseListFilters from './ExpenseListFilters'


const ExpenseDashboardPage=(props)=>(
    <div>
      <EXpenseListFilters />
      <ExpesneList />
      
    </div>
)




export default ExpenseDashboardPage;