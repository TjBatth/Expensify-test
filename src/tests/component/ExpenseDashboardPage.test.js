import React from 'react'
//import ReactShallowRenderer from 'react-test-renderer/shallow'
import {shallow} from 'enzyme'
import toJSON from 'enzyme-to-json';
import ExpenseDashboardPage from '../../component/ExpenseDashboardPage'

test('should NotFoundPage renderer successfully',()=>{
    const wrapper=shallow(<ExpenseDashboardPage />);
    expect(toJSON(wrapper)).toMatchSnapshot();
})
