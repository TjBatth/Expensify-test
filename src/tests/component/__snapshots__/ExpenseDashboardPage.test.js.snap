// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should NotFoundPage renderer successfully 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
